crates/modelardb_storage/src/optimizer/model_simple_aggregates.rs

```diff
@@ -22,7 +22,8 @@
 use std::mem;
 use std::sync::{Arc, LazyLock};
 
-use datafusion::arrow::array::{ArrayRef, BinaryArray, UInt8Array};
+use arrow::array::Int8Array;
+use datafusion::arrow::array::{ArrayRef, BinaryArray};
```

Comment:
Minor but maybe import from either `arrow::array` or `datafusion::arrow::array`.

Reply:
Used `datafusion::arrow::array` for all imports.